etc/helm/pachyderm/templates/pgbouncer/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
+{{- if .Values.pgbouncer.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: pg-bouncer
+    suite: pachyderm
+  name: pg-bouncer
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pg-bouncer
+      suite: pachyderm
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: pg-bouncer
+        suite: pachyderm
+      name: pg-bouncer
+      namespace: default
+    spec:
+      containers:
+      - env:
+        - name: DB_USER
+          value: pachyderm
+        - name: DB_PASSWORD
+          value: elephantastic
+        - name: DB_HOST
+          value: postgres.default
+        - name: AUTH_TYPE
+          value: trust
```

Comment:
I think this needs to be `any` ?

Reply:
Makes sense, I need to plumb through alot of the options